src/components/pages/Add/Add.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import 'firebase/auth';
+import authRequests from '../../../helpers/data/authRequests';
+
+const defaultListing = {
+  city: '',
+  state: '',
+  weather: [],
```

Comment:
I don't think you're using this! Go ahead and take this out

Reply:
deleted weather.